docs/source/api_ref_models.rst

```diff
@@ -2,105 +2,409 @@
 torchtune.models
 ================
 
-.. currentmodule:: torchtune.models
 
-llama3
+This page contains information on all model architectures available in the core torchtune library.
+
+Llama3
 ------
 
-All models from the `Llama3 family <https://llama.meta.com/llama3/>`_.
+.. currentmodule:: torchtune.models.llama3
+
+`Llama 3 <https://llama.meta.com/llama3/>`_ - released in April of 2024 - is an auto-regressive language model that uses an optimized transformer architecture.
+The tuned versions use supervised fine-tuning (SFT) and reinforcement learning with human feedback (RLHF) to align with human preferences for helpfulness and safety.
+The model was pretrained on over **15 trillion tokens** of data from publicly available sources. The pretraining data has a cutoff of March 2023 for the 8B and December 2023
+for the 70B models respectively. The fine-tuning data includes publicly available instruction datasets, as well as over **10M human-annotated examples**.
+You can read more about the model from the official `model card <https://github.com/meta-llama/llama3/blob/main/MODEL_CARD.md#model-details>`_.
+
+Download the model
```

Comment:
I think we've sufficiently covered downloading models in our other docs

Reply:
Having slept on it, I actually like this, particularly if we make the move towards a single page per model family. I think it adds value for users who maybe haven't immediately clocked the usage of `ignore-patterns` or just generally want to get started with fine-tuning a given model without thinking too hard about it.
+1

Reply:
> if we make the move towards a single page per model family

Where are we at on this? I think this would be a nice thing to do